actions/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ environment:
   sdk: ^3.2.0-150.0.dev
 
 dependencies:
```

Comment:
For some reason, the github action was pulling in a very old version of aws_common that was missing a Stream Extension we were relying on. Bumping the version to avoid a slew of "method not supported on Stream" exceptions.